src/Infrastructure/Service/Setting/SettingChoicesProviderRegistry.php

```diff
@@ -48,7 +48,7 @@ public function hasSettingChoicesProvider(string $name): bool
     public function getSettingChoicesProvider(string $name): SettingChoicesProviderInterface
     {
         if (!isset($this->settingChoicesProviders[$name])) {
-            throw new InvalidArgumentException(sprintf('Choice provider %s is not found', $name));
+            throw new InvalidArgumentException(sprintf('Choice provider "%s" is not found', $name));
```

Comment:
Thats not code fencing but quoting, code fencing as per rules is using `

Reply:
Also, isnt it Choices Provider? As per class name?
The choiceProvider typo is a few more times across the PR I see.